server/server.go

```diff
@@ -592,14 +608,12 @@ func (s *server) Health(ctx context.Context, req *rpcpb.HealthRequest) (*rpcpb.H
 		return nil, err
 	}
 
-	s.mu.Lock()
-	defer s.mu.Unlock()
-
-	s.clusterInfo.NodeNames = s.network.nodeNames
-	s.clusterInfo.NodeInfos = s.network.nodeInfos
-	s.clusterInfo.Healthy = true
+	info.NodeNames = s.network.nodeNames
+	info.NodeInfos = s.network.nodeInfos
+	info.Healthy = true
```

Comment:
Also, why does this `Health` function not need to have `CustomChainsHealthy` to have a value? Add this to the documentation.